src/desktop/apps/artwork/client/errorModal.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import { ErrorModal } from "@artsy/reaction/dist/Components/Modal/ErrorModal"
+
+export const render = ({
+  headerText = "An error occurred",
+  detailText = "Something went wrong. Please try again or contact orders@artsy.net.",
+} = {}) => {
+  const el = document.getElementById("react-modal-container")
+  const onClose = () => {
+    ReactDOM.render(<ErrorModal show={false} />, el, () => {
+      ReactDOM.unmountComponentAtNode(el)
+    })
+  }
+  ReactDOM.render(
+    <ErrorModal
+      show={true}
+      onClose={onClose}
```

Comment:
does 
```jsx
onClose={() => ReactDOM.unmountComponentAtNode(el)}
```
not work?

Reply:
I'd had the other render there so that it would animate out... but there's not really an animate out animation anyway, so that works just as well. Hopefully the animation still work on a regular unmount anyway.